README.md

```diff
@@ -21,7 +21,6 @@ interaction with the protocol.
 ## 📓 Docs
 
 - user docs: built from [docs mdBook](./documentation/docs/)
-- dev docs: built from [dev mdBook](./documentation/dev/)
```

Comment:
pls do not remove this - it's being added back in in https://github.com/anoma/namada/pull/266